client/app/(authenticated)/bceidbusiness/industry_user_admin/dashboard/users/page.tsx

```diff
@@ -66,6 +66,7 @@ export default async function Page() {
   ];
 
   const statusRows: GridRowsProp = userOperatorStatuses.map((uOS) => ({
+    id: uOS.id, // This unique ID is needed for DataGrid to work properly
```

Comment:
:+1:
We should also remove lines 14-20 above, showing the guid in the table is not helpful